test/e2e/backup_test.go

```diff
@@ -53,11 +65,14 @@ var _ = Describe("[Restic] Velero tests on cluster using the plugin provider for
 
 	When("kibishii is the sample workload", func() {
 		It("should be successfully backed up and restored to the default BackupStorageLocation", func() {
+			if useVolumeSnapshots && cloudProvider == "kind" {
```

Comment:
Given that both this test and the one below should be performing this check, could we move it into the `BeforeEach` block? That way it would be applied to both, and we could also run the check and skip before we attempt to install Velero.

Reply:
I did that, seems to work, thanks